### DISCUSSION TITLE
For someone new to React and Vue, would you recommend Nuxt or Next?

### DISCUSSION THREAD

**Discussion by saturnonearth** [vercel/next.js#16807] [Help]
I'm new to the scene of Vue and React, and was looking at the differences between Nuxt and Next, what would you guys reccomend?

**Comment by bsantosh909** [comment]
I guess you are basically trying to decided whether to choose **Vue** or **React**.
Because Nuxt and Next are just built on top on their own framework to make things easier like routing, SSR and lots of other stuffs.

So just choose whether to go with Vue or React and then you will get the answer to your own question.

**Reply by saturnonearth** [reply]
I ended up using Vue for the project I had, which was great - I love Vue.js!! For this next project though, sadly I need the power of incremental static regeneration which Nuxt does not currently have.

**Reply by bsantosh909** [reply]
I am not much sure what you are trying to say, but seems like you are going for react them ?? Still as I said you got to go with something you are comfortable with and not what you find others using.. In the end you are the own to be working..

But even while deciding you should always look if the decision meets all your requirements... 


BTW, I looked up for Vue and React initially and decided to go with vue. And to be honest I hardly made a complete project in Vue itself but used Nuxt soon. Because Nuxt makes working with Vue really easy.

**Comment by IRediTOTO** [comment]
:) Yeah, you ask in a NextJS group. You know the answer

**Reply by saturnonearth** [reply]
I actually asked in both groups xD

**Reply by IRediTOTO** [reply]
i like to learn code same as vanilla JS. React is the best option. biggest community, Vue have it ? I pretty sure it cant complete like React. So I choose Nextjs.

**Reply by saturnonearth** [reply]
> i like to learn code same as vanilla JS. React is the best option. biggest community, Vue have it ? I pretty sure it cant complete like React. So I choose Nextjs.

Well just to set the record straight, Vue.js technically speaking has more Stars on github than React, and is entirely community built.
